docs/DeveloperGuide.md

```diff
@@ -319,11 +319,11 @@ and that minimal calculation is needed to reset the bill to 0 (`b/-CURRENT_VALUE
 ### Editing a Guest's Details
 
 In GuestBook, the user can edit the details of a guest using the `edit` command. Details that do not have values inputted in the `edit` command will remain unchanged.
-This feature was implemented so that it is easy for the user to change a guest's details according to different scenarios, such as the changing the guest's rooms, or the guest providing incorrect details.
+This feature was implemented so that it is easy for the user to change a guest's details according to different scenarios, such as the changing of a guest's room, or the guest providing incorrect details.
 
 <span style="font-size: large; color: #e46c0a">Implementation:</span>
-* The `edit` command takes in an `INDEX` indicating the index of the guest to edit in the current panel (starting from 1)
-  and 8 optional fields (`Name`, `Phone`, `Email`, `Room`, `Date Range`, `Number Of Guests`, `Is Room Clean` and `Request`)
+* The `edit` command takes in an `INDEX` indicating the guest to edit in the current panel (starting from 1)
+  and 9 optional fields (`Name`, `Phone`, `Email`, `Room`, `Date Range`, `Number Of Guests`, `Is Room Clean` and `Request`)
```

Comment:
I think there might be 8 optional fields, not 9?

Reply:
Indeed! Not sure how I counted 9, great catch!